source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -47,7 +47,7 @@ class BioXASMainBeamline : public BioXASBeamline
 	{
 		if(!instance_){
 			instance_ = new BioXASMainBeamline();
-			instance_->initializeBeamlineSupport();
+			instance_->initializeBeamline();
```

Comment:
in CLSBeamline, it is more `initializeBeamline()` now than just `initializebeamlineSupport()`.